libs/sdk-core/src/persist/migrations.rs

```diff
@@ -485,6 +485,37 @@ pub(crate) fn current_migrations() -> Vec<&'static str> {
        "ALTER TABLE payments ADD COLUMN is_pseudo INTEGER DEFAULT 0 NOT NULL;
         DELETE FROM payments;
         DELETE FROM cached_items WHERE key = 'sync_state';
+       ",
+       "INSERT INTO cached_items (key, value) VALUES ('current_tip', '0')"
```

Comment:
Is it better to not set a value and return None?

Reply:
You're right. Done. Squashed into the previous commit.